src/main/java/io/github/syst3ms/skriptparser/expressions/ExprContextValue.java

```diff
@@ -0,0 +1,73 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import io.github.syst3ms.skriptparser.registration.ContextValueManager;
+import org.jetbrains.annotations.Nullable;
+
+import static io.github.syst3ms.skriptparser.registration.ContextValueManager.ContextValue;
+
+/**
+ * A specific context value.
+ * Refer to the documentation of the event to see which values can be used.
+ *
+ * @name Context Value
+ * @pattern [the] (past|future|) context-<.+>
+ * @since ALPHA
+ * @author Mwexim
+ */
+public class ExprContextValue implements Expression<Object> {
+
+	static {
+		Main.getMainRegistration().addExpression(
+			ExprContextValue.class,
+			Object.class,
+			false,
+			3,
+			"[the] (1:past|2:future|3:) context-<.+>"
```

Comment:
I'd like to have "previous" and "next" as aliases for this syntax. The third choice is useless as well, the group could just be made optional (`[1:past|2:future]` should work thanks to the added syntax sugar), in which case the default parse mark (and thus the default time value) would be 0. 

Furthermore, I don't know how much I like having `context-` be mandatory, it feels a bit cumbersome. Granted, removing it would require the expression to have very low priority since it essentially becomes `<.+>`, which is bothersome.

Reply:
These changes will all be applied in the next commit except making `context-` optional. I did that and made the priority 1, but even then there were failing tests. It is too 'dangerous' to implement and as a matter of fact not that huge of a deal. We can always revision this later.